web_src/js/modules/fomantic/dropdown.ts

```diff
@@ -74,7 +75,7 @@ function delegateOne($dropdown) {
   // the "template" functions are used for dynamic creation (eg: AJAX)
   const dropdownTemplates = {...dropdownCall('setting', 'templates'), t: performance.now()};
   const dropdownTemplatesMenuOld = dropdownTemplates.menu;
-  dropdownTemplates.menu = function(response, fields, preserveHTML, className) {
+  dropdownTemplates.menu = function(response: any, fields: any, preserveHTML: any, className: string) {
```

Comment:
IIRC className is an object

Reply:
Right, I see I typed it as such in another place.